### ISSUE TITLE
Fatal error: Call to undefined function espresso_display_ticket_selector

### ISSUE THREAD

**Issue by sethshoultes** [eventespresso/eea-beaver-builder#15]
To reproduce, create a new page, then add the EE4 ticket selector to the page using Beaver Builder. Save the page, then try to edit the page using Beaver Builder.

Fatal error: Call to undefined function espresso_display_ticket_selector() in /Users/sethshoultes/Documents/Websites/espresso-4-9.dev.cc/wp-content/plugins/eea-beaver-builder/beaver-builder-modules/ticket-selector/includes/frontend.php on line 4

**Comment by sethshoultes** [comment]
Possibly related From Slack:
@joshfeck saw something similar with elementor
One possible solution would be to tweak the conditionals that determine whether to include the template tags file.
The underlying issue is the page builders are trying to load content as if it was the front end of the site, but it’s not the front end of the site.
Here’s the conditional from EE_System.core.php:
`if (
    is_readable(EE_PUBLIC . 'template_tags.php')
    && ($this->request->isFrontend() || $this->request->isIframe() || $this->request->isFeed())
) {
    require_once EE_PUBLIC . 'template_tags.php';
}`

so one idea could be too look at tweaking what’s defined as `isFrontend()`
and/or add a filter switch to that conditional

**Comment by sethshoultes** [comment]
Here's the possibly related issue for core: https://github.com/eventespresso/event-espresso-core/issues/360

**Comment by sethshoultes** [comment]
Updated: This only seems to be happening in BB 2.1. Earlier versions of BB seem fine.

**Comment by mnelson4** [comment]
Chatted with Simon Prosser in https://beaverbuilders.slack.com and they're removing the following lines from BB 2.1

```
/**
* Dont load ee core, fixes the text editor tabs missing issue.
* @since 2.1
*/
function fl_ee_no_load_builder_active() {
   if ( isset( $_GET['fl_builder'] ) && class_exists( 'EE_System' ) ) {
       remove_action( 'init', array( EE_System::instance(), 'core_loaded_and_ready' ), 9 );
   }
}
add_action( 'after_setup_theme', 'fl_ee_no_load_builder_active' );
```

That was another way to fix issue #13, but accidentally introduced this problem. Once that's released this issue should be resolved too

**Comment by Pross** [comment]
Above code removed as of 2.1.1.1 of Beaver Builder.
